snapshot_manager/snapshot_manager/snapshot_manager.py

```diff
@@ -485,3 +485,256 @@ def handle_labels(
         )
         logging.info(f"Adding label: {labels_to_add}")
         issue.add_to_labels(*labels_to_add)
+
+
+def run_performance_comparison(
+    conf_a: config.Config,
+    conf_b: config.Config,
+    github_repo: str,
+    copr_client: copr.v3.Client,
```

Comment:
The failing test mention this line. Are we missing an `import` statement?

Reply:
Sorry to say this but you should ignore failing tests in the stack if it isn't the top-most PR that you're reviewing.

Reply:
Shouldn't tests pass at every PR in the stack?

Reply:
> Shouldn't tests pass at every PR in the stack?

It depends on what we want. I explained myself in a different comment that I didn't think it was necessary. It helps with git bisect, yes, but it helps less with understanding the "flow" of the commits from the outside to the inside. But since it caused to many questions, I'll never do this again.